extra/grafana/kubernetes/StarRocks-Overview-kubernetes-3.0.json

```diff
@@ -8423,42 +8425,40 @@
         "includeAll": false,
         "label": null,
         "multi": false,
-        "name": "fe_master",
+        "name": "fe_instance",
         "options": [],
         "query": {
-          "query": "label_values(node_info{namespace=\"$namespace\", service=\"$feservice\", type=\"is_master\"}, instance)",
-          "refId": "Prometheus-fe_master-Variable-Query"
+          "query": "label_values(up{namespace=\"$namespace\", service=\"$feservice\"}, instance)",
+          "refId": "Prometheus-fe_instance-Variable-Query"
         },
         "refresh": 1,
         "skipUrlSync": false,
-        "sort": 0,
+        "sort": 1,
         "tagValuesQuery": "",
         "tagsQuery": "",
         "type": "query",
         "useTags": false
       },
       {
-        "allValue": null,
         "current": {
           "selected": false,
           "text": "",
           "value": ""
         },
         "datasource": "${StarRocks_Prometheus}",
         "definition": "",
-        "description": null,
-        "error": null,
         "hide": 0,
         "includeAll": false,
-        "label": null,
+        "label": "service",
         "multi": false,
-        "name": "fe_instance",
+        "name": "service",
```

Comment:
also can make this variable multiple choice including all. so user can select all be/cn services to get a overview of all be/cn instances.